ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -70,21 +83,21 @@ export const notebookProgressStrings: Map<Progress, string> = new Map([
   [Progress.Redirecting, 'Redirecting to the notebook server'],
 ]);
 
-export const terminalProgressStrings: Map<Progress, string> = new Map([
-  [Progress.Unknown, 'Connecting to the terminal'],
-  [Progress.Initializing, 'Initializing terminal, may take up to 5 minutes'],
-  [Progress.Resuming, 'Resuming terminal, may take up to 1 minute'],
-  [Progress.Authenticating, 'Authenticating with the terminal'],
-  [Progress.Copying, 'Copying notebooks onto the server'],
-  [Progress.Creating, 'Opening the terminal'],
-  [Progress.Redirecting, 'Redirecting to the terminal'],
+export const genericProgressStrings: Map<Progress, string> = new Map([
+  [Progress.Unknown, 'Connecting to the environment'],
+  [Progress.Initializing, 'Initializing environment, may take up to 5 minutes'],
+  [Progress.Resuming, 'Resuming environment, may take up to 1 minute'],
+  [Progress.Authenticating, 'Authenticating with the environment'],
+  [Progress.Copying, 'Copying notebooks into the environment'],
+  [Progress.Creating, 'Opening the application'],
+  [Progress.Redirecting, 'Redirecting to the environment'],
 ]);
 
 const getProgressString = (appType: LeoApplicationType, progress: Progress) => {
   if (appType === LeoApplicationType.Notebook) {
     return notebookProgressStrings.get(progress);
```

Comment:
Can this also use genericProgressStrings? Then we can delete The notebookProcess map above

Reply:
I think I prefer to have the detailed messaging for notebooks. This is still the 99% use case for this page, and I think it's reassuring to have the more tailored UX e.g. for creating a new notebook.